### PR TITLE
ROC-2468 Add claimant declaration in ccj details pdf

### PR DESCRIPTION
**Pull request by comanirudha**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-2468

### Change description ###

Add claimant declaration in ccj details pdf

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationService.java [hunk 1]
```diff
@@ -95,6 +96,7 @@ public void logNotificationFailure(
         parameters.put(DEFENDANT_NAME, claim.getClaimData().getDefendant().getName());
         parameters.put(FRONTEND_BASE_URL, notificationsProperties.getFrontendBaseUrl());
         parameters.put(CLAIM_REFERENCE_NUMBER, claim.getReferenceNumber());
+        parameters.put("ccjRequestedDate", Formatting.formatDate(claim.getCountyCourtJudgmentRequestedAt()));
```
Comment (yarpo) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationService.java]: 1. why not const?
2. I'm not sure if the string value is consistent. On the other hand all dates have weird, not-so-consistent names

File: src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationServiceTest.java [hunk 2]
```diff
@@ -42,7 +43,15 @@ public void setup() {
 
     @Test
     public void notifyClaimantShouldCallNotify() throws Exception {
-        Claim claim = SampleClaim.getDefault();
+        Claim claim = SampleClaim.builder()
```
Comment (yarpo) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationServiceTest.java]: why?

Comment (comanirudha) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationServiceTest.java]: need ccj requested date which is not there in default claim.

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationService.java (hunk 1 above)
Comment (comanirudha) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationService.java]: 1. this is the only place this is being used.

File: src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationServiceTest.java [hunk 3]
```diff
@@ -42,7 +43,15 @@ public void setup() {
...
+            .withClaimData(
+                SampleClaimData.builder()
```
Comment (yarpo) [inline src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationServiceTest.java]: but you probably don't need to add specific defendant, do you?

Comment (comanirudha) [reply src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationServiceTest.java]: yes, agree, will remove that

File: src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html [hunk 4]
```diff
@@ -112,5 +112,26 @@
...
 </div>
+<div class="section">
+  <div class="section-heading">
+    <strong>Declaration</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr>
+        <td></td>
+        <td>{{ ccj.claimantName}}</td>
```
Comment (timja-kainos) [inline src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: do we not need the company name and position here?

Comment (comanirudha) [reply src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html]: not as per https://tools.hmcts.net/jira/browse/ROC-2468

((Claimant name)) [in future the person signing the declaration could be someone other than the claimant, but for now it's only this option]

**Pull request merged**